### PR TITLE
Adelle/fix reload loop

### PR DESCRIPTION
**Pull request by Adelle-Pitsas**
See notes in issue:
Xref issue: https://github.com/gulfofmaine/Neracoos-1-Buoy-App/issues/2651

Fixes rerendering loop by adding a suspense boundary and forcing back to client-side

### CODE REVIEW

File: app/(modelsMap)/models/modeling.tsx [hunk 1]
```diff
@@ -12,6 +14,31 @@ import { useLatestItemsByCollectionIdsQuery, useRootCatalogQuery } from "./stac-
...
+  }, [])
+
+  return (
+    <>
+      <Row>
+        <Col md={3} style={{ padding: 0 }}>
+          <StacCatalogRoot />
+        </Col>
+        <Col md={9} style={{ paddingLeft: 0 }}>
+          {!loading && <StacMap />}
```
Comment (abkfenris) [inline app/(modelsMap)/models/modeling.tsx:31]: Is this pattern to delay drawing the map until it's on the client?

**Review by abkfenris — APPROVED** [review]
This seems to work when I tested it locally. I'm wondering about the `useState` `useEffect` pattern you have, and what that's supposed to be doing?

File: app/(modelsMap)/models/modeling.tsx (hunk 1 above)
Comment (Adelle-Pitsas) [reply app/(modelsMap)/models/modeling.tsx:31]: Yes, thats correct

Comment (abkfenris) [reply app/(modelsMap)/models/modeling.tsx:31]: Ah, missed your comment on the issue. Feel free to merge when you are ready.

**Pull request merged**